src/prometheus_alert_rules/loki_process_too_many_restarts.rule

```diff
@@ -0,0 +1,10 @@
+# Based on https://awesome-prometheus-alerts.grep.to/rules.html#loki
+# Given the way Loki charm behaves seems reasonable to warn if Loki restarts 5 times in 15 minutes
```

Comment:
```suggestion
# The Loki charm may restart the workload on config-changed event, which means there would be
# two restarts just from deploying the charm.
# Changing the config soon after deploy (e.g. set resource limits) would cause another restart.
# Using 5 as the alert threshold to take into account restart (2 starts) + immediate config change (1 more start).
```

However, seems like `process_start_time_seconds` is not a restart count per se. Does "5" mean loki spent 5 seconds starting?

Reply:
Nop, `process_start_time_seconds` it is not a count per se. What transforms it in a count is the `changes` function:

```
changes(process_start_time_seconds[15m]) > 5
```

The result of this would be:

![imagen](https://user-images.githubusercontent.com/939888/177394274-51498af1-b676-4d37-b508-d33842ff30b0.png)